Android/ConstraintLayout/build.cake

```diff
@@ -1,35 +1,65 @@
-#load "../../common.cake"
+var TARGET = Argument ("target", Argument ("t", "ci"));
 
-var TARGET = Argument ("target", Argument ("t", "Default"));
-
-var NUGET_VERSION = "2.0.0-alpha2";
-var AAR_VERSION = "2.0.0-alpha2";
+var NUGET_VERSION = "2.0.0-beta2";
+var AAR_VERSION = "2.0.0-beta2";
```

Comment:
I see the version numbers haven't changed, did we just not release it?

Reply:
-alpha2 vs -beta2...